GirafRest.Test/Controllers/UserControllerTest2021.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using GirafRest.Controllers;
+using System.Text;
+using Xunit;
+using Moq;
+using Microsoft.AspNetCore.Http;
+using Microsoft.EntityFrameworkCore;
+using GirafRest.Extensions;
+using GirafRest.Models;
+using GirafRest.Models.DTOs;
+using GirafRest.Models.Responses;
+using GirafRest.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore.InMemory;
+using GirafRest.Data;
+using GirafRest.Test.Mocks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace GirafRest.Test.Controllers
+{
+    [TestClass]
+    public class UserControllerTest2021
+    {
+        readonly GirafRoles citizen = GirafRoles.Citizen;
+
+        [TestMethod]
+        public void GetSortedCitizens()
+        {
+            //Arrange 
+            List<DisplayNameDTO> unsorted_test_list = new List<DisplayNameDTO>();
+            List<DisplayNameDTO> sorted_test_list = new List<DisplayNameDTO>();
+            DisplayNameDTO Kenneth = new DisplayNameDTO("Kenneth", citizen, "1101");
+            DisplayNameDTO Louise = new DisplayNameDTO("Louise", citizen, "1101");
+            DisplayNameDTO Josefine = new DisplayNameDTO("Josefine", citizen, "1101");
+            DisplayNameDTO Søren = new DisplayNameDTO("Søren", citizen, "1101");
```

Comment:
Roslyn compileren bruger unicode, men jeg tror linting tools og andet statisk analyse, kun bruger ASCII. Jeg vil derfor mene at det vil være bedst at bruge identifieren ```Soeren``` i stedet for ```Søren```.